pkgs/applications/audio/curseradio/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3Packages, mpv }:
+
+python3Packages.buildPythonApplication rec {
+  version = "0.2";
+  name    = "curseradio-${version}";
```

```suggestion
  pname    = "curseradio";
```

`pname` should always be used in python packages.

---

Btw, `pname` and `version` just get combined with a `-` to make the derivation name.